src/twyn/config/config_handler.py

```diff
@@ -47,8 +47,9 @@ class ReadTwynConfiguration:
 class ConfigHandler:
     """Manage reading and writing configurations for Twyn."""
 
-    def __init__(self, file_path: Optional[str] = None, enforce_file: bool = True):
-        self._file_path = file_path or DEFAULT_PROJECT_TOML_FILE
+    def __init__(self, file_path: Optional[str] = None, enforce_file: bool = True) -> None:
+        file_path = file_path or DEFAULT_PROJECT_TOML_FILE
+        self.file_handler = FileHandler(file_path)
```

Comment:
Instead of passing a file_path and constructing the `FileHandler` inside the init, I would 100% pass a `FileHandler` instance.

Reply:
```python
def __init__(self, file_handler: Optional[BaseFileHandler] = None, enforce_file: bool = True) -> None:
    self.file_handler = file_handler or FileHandler(file_path) # `FileHandler` is now just a default file handler, but you could pass any you may want
```

Reply:
I've made it so it receives a `FileHandler` object, which is mandatory.